libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -453,7 +450,7 @@ namespace eosio { namespace testing {
    class tester : public base_tester {
    public:
       tester(setup_policy policy = setup_policy::full, db_read_mode read_mode = db_read_mode::HEAD, std::optional<uint32_t> genesis_max_inline_action_size = std::optional<uint32_t>{}, std::optional<uint32_t> config_max_nonprivileged_inline_action_size = std::optional<uint32_t>{}) {
```

Comment:
should you remove the parameter as well, or maybe change the name to `unused`.
`std::optional<uint32_t> config_max_nonprivileged_inline_action_size = std::optional<uint32_t>{}`

Reply:
Thanks, not sure how my search missed that.